packages/ui/src/hooks/use-clipboard-otp.ts

```
@@ -0,0 +1,222 @@
+      }
+
+      return "";
+    },
+    [otpLength, pattern],
+  );
+
+  const checkPermissions = useCallback(async (): Promise<boolean> => {
+    if (!isSupported) return false;
+
+    try {
+      if (navigator.permissions && navigator.permissions.query) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)